development.md

```diff
@@ -0,0 +1,91 @@
+# Swift Driver Development Guide
+
+## Index
+* [Things to install](#things-to-install) 
+* [The code](#the-code)
+* [Building](#building)
+* [Running tests](#running-tests)
```

Comment:
Title capitalization:

 * Things to Install
 * The Code
 * Running Tests

Reply:
👍